public/app/features/scenes/variables/variants/MultiValueVariable.test.ts

```diff
@@ -35,6 +35,24 @@ describe('MultiValueVariable', () => {
       expect(variable.state.text).toBe('B');
     });
 
+    it('Should pick All value when defaultToAll is true', async () => {
+      const variable = new ExampleVariable({
+        name: 'test',
+        options: [],
+        optionsToReturn: [
+          { label: 'B', value: 'B' },
+          { label: 'C', value: 'C' },
+        ],
+        defaultToAll: true,
+        value: 'A',
+        text: 'A',
```

Comment:
Nit: I would remove this from the test to make it clear that it tests defaultToAll and not default value when the provided value is not a valid option. I think this case is tested on line 109